doc/modules/kernel_approximation.rst

```diff
@@ -149,6 +149,28 @@ above for the :class:`RBFSampler`. The only difference is in the free
 parameter, that is called :math:`c`.
 For a motivation for this mapping and the mathematical details see [LS2010]_.
 
+TensorSketch Method for Polynomial Kernel Approximation
+--------------------------------------------------------
+
+The TensorSketch [PP2013]_ method, as implemented in :class:`PolynomialSampler`, is a
+scalable, data-independent method for polynomial kernel approximation. This method
+obtains a Count Sketch [CCF2002]_ of the outer product of two vectors (or a vector with
```

Comment:
See my comment above.

Reply:
Maybe 
"""
Count sketch is a dimensionality reduction technique similar to feature hashing, but that uses a several independent hash functions.
"""
to paraphrase https://en.wikipedia.org/wiki/Count_sketch Actually it could be useful to additionally have this link in the text. The original paper is good but it's still simpler to read on wiki for a casual reader.